devices/ikea/vallhorn_wireless_motion_sensor.json

```diff
@@ -239,7 +243,8 @@
             "cl": "0x0400",
             "at": "0x0000",
             "script": "0400_lightlevel.js"
-          }
+          },
+          "refresh.interval": 86400

```

Comment:
Could you please remove the refresh interval again? It isn't considered anyway here and is therefore superfluous. And please do not forget the comma above 😉